404 Not Found: @generator/library@^0.1.0

```
» npm i                                                                                                                                                    1 ↵
npm ERR! code E404
npm ERR! 404 Not Found: @generator/library@^0.1.0

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dwade/.npm/_logs/2018-12-09T00_45_47_267Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "@mbunifu/generator",
   "author": "Doug Wade <douglas.b.wade@gmail.com>",
   "license": "MIT",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "description": "Generator for project generators. Meta!",
   "dependencies": {
     "@mbunifu/library": "^0.0.1",

--- a/templates/generator/README.md
+++ b/templates/generator/README.md
@@ -33,7 +33,7 @@ npm install --save @generator/<% name %>
 Then require it and run the `run` command
 
 ```js
-const <% pascalCaseName %> = require('@generator/my-generator');
+const <% pascalCaseName %> = require('@mbunifu/my-generator');
 
 const <% camelCaseName %> = new <% pascalCaseName %>();
 
@@ -43,7 +43,7 @@ const <% camelCaseName %> = new <% pascalCaseName %>();
 You can provide any of the accepted arguments as a named argument to the constructor.
 
 ```js
-const <% pascalCaseName %> = require('@generator/my-generator');
+const <% pascalCaseName %> = require('@mbunifu/my-generator');
 
 const <% camelCaseName %> = new <% pascalCaseName %>({
     dryRun: true,
@@ -64,6 +64,6 @@ The new project is now located at `./my-new-project`.
 
 # Find out more
 
-This generator is powered by `@generator/library`.  Find out more about how it works and
-how to make your own by reading its [README.md](https://github.com/doug-wade/generator-library)
-or its [annotate source code](https://github.com/doug-wade/generator-library).
+This generator is powered by `@mbunifu/library`.  Find out more about how it works and
+how to make your own by reading its [README.md](https://github.com/doug-wade/mbunifu-library)
+or its [annotate source code](https://github.com/doug-wade/mbunifu-library).

--- a/templates/generator/package.json
+++ b/templates/generator/package.json
@@ -5,7 +5,7 @@
   "version": "0.0.1",
   "description": "Generator for projects.",
   "dependencies": {
-    "@generator/library": "^0.1.0",
+    "@mbunifu/library": "^0.1.0",
     "inquirer": "^5.2.0"
   },
   "scripts": {
@@ -23,7 +23,7 @@
     "eslint-plugin-react": "^6.10.3",
     "jest": "^22.4.3"
   },
-  "bin": "node_modules/@generator/library/bin/index.js",
+  "bin": "node_modules/@mbunifu/library/bin/index.js",
   "engines": {
     "node": ">=8.9",
     "npm": ">=5.5.1"

